pool/acceptedwork_test.go

```diff
@@ -93,27 +93,14 @@ func testAcceptedWork(t *testing.T, db *bolt.DB) {
 			fetchedWork.Height, workC.Height)
 	}
 
-	// Ensure requesting negative or zero most recent accepted work returns
-	// an error.
-	_, err = listMinedWorkByAccount(db, string(id), -1)
```

Comment:
Can we keep `listMinedWorkByAccount` around just for these tests?

Reply:
`listMinedWorkByAccount` was called three times in test code.

Two of these appearances were just testing the behaviour of `listMinedWorkByAccount` itself, so I wont bother to reinstate those.

The third call was to ensure mined work is persisted in the database correctly, so this one seems worth reinstating. I will put back some code which performs this check.